src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -170,12 +171,34 @@ else if(messageContent != null) {
 
       messageStore.addMessage(message);
 
-      List<String> messageInformation = new ArrayList<>();
+      List<String> messageInformation = new ArrayList<String>();
       messageInformation.add(user.getName());
       messageInformation.add(conversationTitle);
       messageInformation.add(cleanedMessageContent);
       Event messageEvent = new Event(UUID.randomUUID(), "Message", message.getCreationTime(), messageInformation);
       eventStore.addEvent(messageEvent);
+
+      // Scan the message for "@NemoBot"
+      if (cleanedMessageContent.toLowerCase().contains("@NemoBot".toLowerCase())) {
+        Bot chatBot = new Bot();
+        String botResponse = chatBot.parseMessage(cleanedMessageContent);
+        Message botMessage =
+            new Message(
+                UUID.randomUUID(),
+                conversation.getId(),
+                chatBot.getId(),
+                botResponse,
+                Instant.now());
+
+        messageStore.addMessage(botMessage);
+
+        List<String> botMessageInformation = new ArrayList<String>();
+        botMessageInformation.add("NemoBot");
+        botMessageInformation.add(conversationTitle);
+        botMessageInformation.add(botResponse);
+        Event botMessageEvent = new Event(UUID.randomUUID(), "Message", botMessage.getCreationTime(), botMessageInformation);
```

Comment:
nit: can we break up long lines to help with readability?

Reply:
Will do, sorry about that.